android/gradle.properties

```diff
@@ -1,3 +1,7 @@
-Apptentive_kotlinVersion=1.5.20
-Apptentive_compileSdkVersion=31
-Apptentive_targetSdkVersion=31
+Apptentive_kotlinVersion=1.7.20
+Apptentive_minSdkVersion=21
```

Comment:
Need to verify the behavior if the Apps are in different compileSDKVersion, as we allow from 31 -33

Reply:
Oh yeah... I could (and maybe should) go down to 31.